src/ResourceManager/EventHub/ChangeLog.md

```diff
@@ -19,7 +19,26 @@
 -->
 ## Current Release
 
-## Version 0.4.2
+## Version 4.3.0
```

Comment:
@v-Ajnava please move the content below to under the `Current Release` header, and revert the changes made to `Version 0.4.2` header